### DISCUSSION TITLE
How do I direct traffic out of gluetun, but open local ports for local access?

### DISCUSSION THREAD

**Discussion by ForceConstant** [qdm12/gluetun#740] [Q&A]
I know how to setup gluetun using the network_mode to direct towards gluetun container. But docker won't let me do this and also have local ports open, i.e. set with the ports: or -p options. So what is the right way to do this? I am guessing that if I use docker-compose create a specific gluetun server for this other service, and then put these local ports in the gluetun section?

**Comment by qdm12** [comment]
Publish the ports on gluetun, since it's the only network host Docker - speaking. More details at https://github.com/qdm12/gluetun/wiki/Port-mapping
